modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -756,11 +755,9 @@ else if (Objects.equals(
 			ObjectValidationRuleSettingConstants.
 				NAME_COMPOSITE_KEY_OBJECT_FIELD_ID);
 
-		if (FeatureFlagManagerUtil.isEnabled("LPD-29637")) {
-			allowedObjectValidationRuleSettingNames.add(
-				ObjectValidationRuleSettingConstants.
-					NAME_ALLOW_ACTIVE_STATUS_UPDATE);
-		}
+		allowedObjectValidationRuleSettingNames.add(
+			ObjectValidationRuleSettingConstants.
+				NAME_ALLOW_ACTIVE_STATUS_UPDATE);
```

Comment:
Can you add the constant to the allowedObjectValidationRuleSettingNames? sort it, please.